tests/kuttl/common/osp_check_noapi_service_certs.sh

```diff
@@ -56,6 +56,7 @@ done
 
 # Gather the ClusterIP and ports for databases
 for database in "${!database_secrets[@]}"; do
+    sleep 5
```

Comment:
just wondering whats different to the db checks then to the above services. I mean should we have those sleeps also in the above?

Reply:
right, maybe here it is not needed - it should handle.
Next steps - iterating on ports is most problematic.